Xamarin.Forms.Platform.UAP/DatePickerRenderer.cs

```diff
@@ -104,6 +106,8 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				UpdateMaximumDate();
 			else if (e.PropertyName == DatePicker.MinimumDateProperty.PropertyName)
 				UpdateMinimumDate();
+			else if (e.PropertyName == VisualElement.BackgroundProperty.PropertyName)
```

Comment:
Isn't this call already taken care of by the base class?

Reply:
Right. This is unnecessary. I reviewed and removed some places updating the brush.